sample/QueueTrigger-Powershell/function.json

```diff
@@ -1,17 +1,19 @@
-﻿{
-    "bindings": [
-        {
-            "type": "queueTrigger",
-            "direction": "in",
-            "queueName": "samples-powershell"
-        },
-        {
-            "type": "table",
-            "name": "output",
-            "direction": "out",
-            "tableName": "samples",
-            "partitionKey": "samples-powershell",
-            "rowKey": "{id}"
-        }
-    ]
-}
+﻿{
+  "bindings": [

```

Comment:
Also, she did add explicit authlevel and disabled.

Cool trick: when looking at CR, add `?w=1` to URL to ignore spaces. Then the change becomes clearer.